openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -230,7 +230,9 @@ The following requirements apply for both, the Wallet and the Verifier, unless s
   * The Wallet MUST support both signed and unsigned requests as defined in Annex A.3.1 and A.3.2 of [@!OIDF.OID4VP]. The Verifier MAY support signed requests, unsigned requests, or both.
 * Wallet Invocation is done via the W3C Digital Credentials API or an equivalent platform API. Any other mechanism, including Custom URL schemes, MUST NOT be used.
 * Response Mode MUST be `dc_api.jwt`. The response MUST be encrypted.
-* Specific requirements for the response encryption are tbd (https://github.com/openid/oid4vc-haip/issues/131).
+* Response encryption MUST be performed as specified in [@!OIDF.OID4VP, section 7.3]. The JWE `alg` (algorithm) header parameter (see [@!RFC7516, section 4.1.1])
+  value MUST be `ECDH-ES` (as defined in [@!RFC7518, section 4.6]), with key agreement utilizing keys on the `P-256` curve (see [@!RFC7518, section 6.2.1.1]).
+  The JWE `enc` (encryption algorithm) header parameter (see [@!RFC7516, section 4.1.2]) value MUST be `A128GCM` (as defined in [@!RFC7518, section 5.3]).
```

Comment:
Maybe this text would be better? It wouldn't preclude the use of the great and powerful HPKE should the https://datatracker.ietf.org/doc/draft-ietf-jose-hpke-encrypt/ endeavor pan out into something reasonable. But gives an MTI now that encourages interop. 

```suggestion
* Response encryption MUST be performed as specified in [@!OIDF.OID4VP, section 7.3]. The JWE `alg` (algorithm) header parameter (see [@!RFC7516, section 4.1.1])
  value `ECDH-ES` (as defined in [@!RFC7518, section 4.6]), with key agreement utilizing keys on the `P-256` curve (see [@!RFC7518, section 6.2.1.1]) MUST be supported.
  The JWE `enc` (encryption algorithm) header parameter (see [@!RFC7516, section 4.1.2]) value `A128GCM` (as defined in [@!RFC7518, section 5.3]) MUST be supported.
```